includes/Extension/SearchReplace/class-replace.php

```diff
@@ -16,11 +16,14 @@ class Replace {
 	public function __construct() {
 		add_action( 'string_locator_search_results_tablenav_controls', array( $this, 'add_replace_button' ) );
 		add_action( 'string_locator_search_results_tablenav_controls', array( $this, 'output_replace_form' ) );
+		add_action( 'string_locator_instawp_tablenav_controls', array( $this, 'add_instawp_stage_button' ) );
 
 		add_action( 'admin_enqueue_scripts', array( $this, 'maybe_enqueue_assets' ) );
 
-		add_action( 'string_locator_search_templates', array( $this, 'add_replace_response_template' ) );
-	}
+		add_action( 'string_locator_search_templates', array( $this, 'add_replace_response_template' ) ); 
+		
+		add_action('wp_ajax_install_activate_plugin', array($this, 'install_activate_plugin_callback'));
```

Comment:
This section need some spacing to match the WordPress coding standards

```suggestion
		add_action( 'wp_ajax_install_activate_plugin', array( $this, 'install_activate_plugin_callback' ) );
```

Reply:
Resolved In latest Commit.